modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -229,12 +230,13 @@ const setup = (editor: Editor): RenderInfo => {
         tag: 'div',
         classes: [ 'tox', 'tox-silver-sink', 'tox-tinymce-aux' ].concat(deviceClasses),
         attributes: {
-          ...I18n.isRtl() ? { dir: 'rtl' } : {}
+          ...I18n.isRtl() ? { dir: 'rtl' } : {},
+          style: isInShadowRoot ? 'top: 0; left: 0; right: 0; position: fixed' : ''
```

Comment:
Alloy should have a way to do this already, it shouldn't need custom styles?

Reply:
I couldn't find anything that sets styles to position fixed and 100% viewport width. The isFixed either sets position relative or no styles at all. Maybe I missed something.

Reply:
I think what Andy means is that alloy doesn't tend to use `attribute` to set `style`.

Reply:
Actually, no .. that's not what he meant ... but it's what I mean :)

Reply:
Hmm. I thought it would set position: fixed, but maybe it just assumed that the integrator would do that. I'll check.

Yeah, looks like it's been that way for ages.